### ISSUE TITLE
Can you share some details on how to generate the miniImageNet pickle files, I want to test on my own datasets, many thanks.

### ISSUE THREAD

**Issue by LHCGB12** [gidariss/FewShotWithoutForgetting#10]

**Comment by gidariss** [comment]
Yes. I will clean the corresponding code and I will upload it. I will let you know when is ready.

**Comment by LHCGB12** [comment]
thanks a lot!

**Comment by LHCGB12** [comment]
Hi, sorry to bother you before Christmas, but I have some problems generating the pickle files myself, if the code will not be ready shortly, can you just share some raw sample codes on how to label the data, thanks.

**Comment by qinhui1003** [comment]
The pickle file can't be downloaded in China, Could you upload the codes how to generate it? thanks!

**Comment by EmmanouelP** [comment]
Do we have any news on that sirs?

**Comment by EternalWang** [comment]
> Yes. I will clean the corresponding code and I will upload it. I will let you know when is ready.

8 months passed, but we still haven't seen the code.
